client/src/components/Donation/DonationTextComponents.js

```diff
@@ -28,33 +28,26 @@ export const DonationOptionsText = () => (
   <>
     <h4>
       <b>
-        Want to make a bigger one-time donation, mail us a check, or give in
-        other ways?
+        <Trans>donate.bigger-donation</Trans>
       </b>
     </h4>
     <p>
-      Here are many{' '}
-      <a
-        href={
-          'https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp'
-        }
-      >
-        other ways we could support our non-profit's mission
-      </a>
-      .
+      <Trans i18nKey='donate.other-ways'>
+        <a href='https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp'>
+          placeholder
+        </a>
+      </Trans>
     </p>
   </>
 );
 
 export const DonationOptionsAlertText = () => (
   <p>
-    Want to make a bigger one-time donation, mail us a check, or give in other
-    ways? Here are many{' '}
-    <a
-      href={'https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp'}
-    >
-      other ways we could support our non-profit's mission
-    </a>
-    .
+    <Trans>donate.bigger-donation</Trans>{' '}
+    <Trans i18nKey='donate.other-ways'>
+      <a href='https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp'>
```

Comment:
Should this link be translatable too? With the active effort to translate `/news` articles, being able to replace this ref with the correct translated article would be important IMHO.

Reply:
That's a great question. I have created https://github.com/freeCodeCamp/freeCodeCamp/issues/40588, to discuss this further.